bower.json

```diff
@@ -17,7 +17,8 @@
     "bootstrap": ">=3.1.0 <4",
     "moment": ">=2.6.0",
     "jquery-extendext": ">=0.1.2",
-    "doT": ">=1.0.3"
+    "doT": ">=1.0.3",
+    "chosen": "^1.8.3"
```

Comment:
it cannot be an explicit dependency, move this to examples/bower.json

Reply:
Yes. Have moved this to examples/bower.json